options.go

```diff
@@ -18,6 +19,9 @@ type Options struct {
 	// Default value is 5.
 	MemtableNums int
 
+	// SegmentSize specifies the maximum size of each segment file in bytes.
+	SegmentSize int64
```

Comment:
I think this not belongs to the value log test, you can create another PR for this